tasks/qiime-demultiplexing/main.py

```diff
@@ -67,6 +67,9 @@ def main() -> None:
 
         raise ValueError(">> [Qiime: Demux] Dataset has 0 imported samples")
 
+    if taskRun.parameters["barcodeColumn"] is None:
```

Comment:
Use `dict.get` to cover the case where a key doesn't exist. When using `[]` operator to access values in dictionary if a key doesn't exist an exception will be raised whereas using `dict.get` will return `None` if a key doesn't exist.